### ISSUE TITLE
[Bug] Minor error when using -m

### ISSUE THREAD

**Issue by verafide** [althonos/InstaLooter#59]
`G:\prattprattpratt>instaLooter -m -n 5 prattprattpratt
Downloading |  0%(0 of 5)|               |Elapsed Time: 0:00:01 |ETA:  --:--:--7
:58:20 - WARNING - unclosed file <_io.BufferedReader name='G:\\prattprattpratt\\
1490604707919288986.jpg'>`

"WARNING - unclosed file <_io.BufferedReader" The files still download so I'm not sure it's actually causing any problems. This first started happening after the latest update.

**Comment by althonos** [comment]
No idea what caused the warning to start appearing since I did not change parts of code dealing with I/O. This is more likely an issue with the formatting of warnings, but I'll check where I need some fixes to remove the warning entirerly.

**Comment by althonos** [comment]
@verafide: Can't reproduce this on Linux, does it still happens on Windows ?

**Comment by verafide** [comment]
@althonos Yeah it's still happening on Windows. When I exclude `-m`  the warning doesn't appear so I guess it's a problem there.

**Comment by verafide** [comment]
@althonos Yeah it's still happening on Windows. This first started on version 0.9.0 I've now rolled back a version to see if the error still appears and it's gone when on 0.8.2

**Comment by prohtex** [comment]
I'm also getting the "unclosed file" error on MacOS

**Comment by prohtex** [comment]
Still happening with 0.11.0

**Comment by althonos** [comment]
I just realized these warnings may be raised by `requests.Response` objects, since thos were the only ones I was not closing properly. I added context managers where applicable, so could any of you try installing the development version from GitHub and tell me if the warnings are gone ? 
```
pip install --user --upgrade https://github.com/althonos/InstaLooter/archive/master.zip
```

**Comment by prohtex** [comment]
That did not fix the issue for me
